Branch-SDK/src/io/branch/indexing/ContentDiscoverer.java

```diff
@@ -166,8 +168,8 @@ public void run() {
                         // Cache the analytics data for future use
                         PrefHelper.getInstance(activity).saveBranchAnalyticsData(contentEvent_);
 
-                        if (cdManifest_.getDiscoveryRepeatTime() != -1) {
-                            handler_.postDelayed(readContentRunnable, cdManifest_.getDiscoveryRepeatTime());
+                        if (cdManifest_.getCDPathProperties(activity).getDiscoveryRepeatTime() != DRT_NOT_SET) {
```

Comment:
Can we avoid calling `getDiscoveryRepeatTime()` twice if the condition is `true`.  This method involves Json parsing